articles/Q-language setup.md

```diff
@@ -0,0 +1,70 @@
+# Setup and workflow for Q# language topics from qsharp-language repo
+
+This document is to define the setup and workflow for syncing the Q# language topics in the qsharp-language repo to the quantum-docs-pr repo for publishing as part of the QDK docset. 
+
+* [Quantum-docs-pr structure](#quantum-docs-pr-structure)
+* [QDK TOC structure](#qdk-toc-structure)
+* [File sync process](#file-sync-process)
+* [Stubbed topics in quantum-docs-pr](#stubbed-topics-in-quantum-docs-pr)
+* [Source topics](#source-topics)
+
+Syntax for the Markdown *include* statement is [here](https://review.docs.microsoft.com/help/contribute/includes-best-practices?branch=master).
+
+**Overall workflow:**
+
+1. Source content is created in qsharp-language repo.
+1. Source content files are copied to the /includes folder (and sub-folders) in quantum-docs-pr repo.
```

Comment:
Did it work to use a reference to the qsharp-language repo or a submodule? I worry about keeping copies in sync manually. At the worst, if we do need to copy and paste files, I'd like to make sure we have a GitHub Action workflow on the language repo that makes a PR on this repo to do that copying-and-pasting, with a warning that any changes here will be overwritten.

Reply:
As an update, I was able to get including as a dependent repository to work without copying-and-pasting: https://review.docs.microsoft.com/en-us/quantum/user-guide/language/types?branch=cgranade%2Fexperimental%2Flang-dep

Reply:
Oh, that's very cool. I had to do some reading as I've never used it before, but that's great.  I wasn't happy with the manual process, either - thanks!

Reply:
No worries, happy to help!